test/nashorn/script/basic/JDK-8283339.js.EXPECTED

```diff
@@ -0,0 +1 @@
+test
```

Comment:
Please add a newline at the end of file. It's weird the test passes since `print(a)` on line 34 in the test does emit a newline.

Reply:
Done, thanks for the review!